Brainfuck/spark/alire.toml

```diff
@@ -0,0 +1,11 @@
+name = "spark"
```

Comment:
None of the other ports make use of package managers. Can we do this without package managers?

Reply:
If that means dropping `gnatprove` then maybe we can just leave that as a note in program.yaml.

I've struggled that while it's useful to also see a non-trivial project (such as using more than one file and bringing in a dependency), I don't really want that complexity for now across the board.